### PR TITLE
New callout when there aren´t agents available

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team! I have added a callout when there aren´t any agents that shows the view to add an agent.
This closes [#2564](https://github.com/wazuh/wazuh-kibana-app/issues/2564)
Captura de pantalla 2020-11-18 a las 13 25 57
Captura de pantalla 2020-11-18 a las 13 26 36

### CODE REVIEW

File: public/components/common/welcome/overview-welcome.js [hunk 1]
```diff
@@ -66,12 +67,26 @@ export class OverviewWelcome extends Component {
...
+          <EuiCallOut   title="No agents were added to this manager. " color="warning" iconType="alert">
+            <EuiButtonEmpty style={{margin: "-58px 286px"}} alignItems='right' color="warning" href='#/agents-preview?'>Add agent</EuiButtonEmpty>
+          </EuiCallOut>
+        </EuiFlexItem >
+      </EuiFlexGroup>
+    );
+  }
+
   render() {
+    console.log("erwe", this.props)
```
Comment (frankeros) [inline public/components/common/welcome/overview-welcome.js:83]: remove console.log

**Review by frankeros — CHANGES_REQUESTED** [review]

File: public/components/common/welcome/overview-welcome.js (hunk 1 above)
Comment (pablomarga) [reply public/components/common/welcome/overview-welcome.js]: Removed, thx.

**Comment by Desvelao** [comment]
Maybe, you could add vertical space between the callout and the panel?

**Review by jsanchez91 — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**